src/test/ui/did_you_mean/bad-assoc-expr.rs

```diff
@@ -21,8 +21,8 @@ fn main() {
 
 macro_rules! expr {
     ($ty: ty) => ($ty::clone(&0))
-    //~^ ERROR missing angle brackets in associated item path
```

Comment:
This *should* be an error (at least with the current path grammar).
It doesn't happen because the parser drops `Invisible` delimiters and turns this into `u8::clone(&0)` instead of `«u8»::clone(&0)`.